### ISSUE TITLE
statement info box doesn't show up

### ISSUE THREAD

**Issue by ltjansen** [leifeld/dna#119]
Hi, I'm currently working on my master's thesis using DNA, but I've encountered a problem that keeps me from continuing. Working in a document with now a hundred statements, new statements that I create won't show the usual box in which you can fill out the person/organisation/concept/agreement information. Clicking on previous statements, the box shows up just like it should. The issue only seems to apply to this particular document at the moment, as I've tested marking statements in a new one, and there it worked just fine. Annotation info boxes also work. 

I am using DNA version 2.0 beta 19
Hopefully this issue can be resolved!

**Comment by JBGruber** [comment]
Hi,

Can you please provide a few more details about the issue:

1. When you select text and then right-click, does the `Format as DNA Statement` option appear? 
2. Can you still create DNA Statements and do they show up in the Statements box in the upper right of the main window?
3. Are you using multiple coders in your project? 
4. Does your coder have all permissions (click on small pencil icon coder box, upper right corner of DNA to see)?
5. What happens if you add a second coder, switch to that coder and try to add a statement?

Also: Have you noticed that there is a [new beta?](https://github.com/leifeld/dna/releases/tag/v2.0-beta.20) It probably won't solve your problem to use the newer version but it also won't hurt.

**Comment by ltjansen** [comment]
Thank you for your quick response. 
(1) the 'format as DNA statement' option does appear
(2) when clicking this option, a new statement appears in the upper right box
(3) I am the only coder
(4) from the start, all permissions were set
(5) after adding a second 'test-coder' the issue remained the same

Just as an afterthought, it is possible to code more than a hundred statements in the same DNA file right (divided among multiple documents)? 

Thank you in advance

**Comment by JBGruber** [comment]
This sounds like the same bug that is described in #103 then. By following the steps in this comment https://github.com/leifeld/dna/issues/103#issuecomment-330766820, you should be able to repair your dataset.

We would appreciate it if you could describe as closely as possible what you did last before this behaviour of DNA started. It seems to be a bug in DNA but we don't know how to reproduce the issue.

Let us know if the fix worked for you, thanks.

**Comment by ltjansen** [comment]
I followed the steps described in the #103 thread, and it worked for me. Thank you! As I recall I have been cleaning up my list of concepts, sometimes deleting ones that I didn't use anymore. I must have triggered this problem when removing a 'blank' entry, but I'm not sure how that would have happened.

I did this in the same attribute window (located at the bottom of the screen), using the middle tab (the one with paper and pencil). Here I removed the blank entry in my list of concepts, after checking all statements actually had a concept assigned to them (there was actually one that I forgot). However, this seems to me like a while back, and  between that time and the time this problem occured coding statements worked just fine. I'm sorry I cannot provide more exact information on this process. 

Now that I know how to fix this issue I can go back to coding, so thank you again.
